mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
+python_version = 3.6
+namespace_packages = True
+ignore_missing_imports = True
```

Comment:
Shouldn't need this one:  we would prefer to add explicit `# type: ignore` for imports, so that we can remove them later when the dependency adds typing.